### PR TITLE
Add configurable colormap

### PR DESCRIPTION
**Pull request by riecke**
Summary of changes:

- Create several accessible colormaps (viridis, inferno, wistia) along with ones for PostGIS ST_ColorMap in the Django backend
- Wire the "styles" parameter for WMS REST queries to select one of these colormaps (default viridis)
- Add a "colormap" to the conditions.json file
- Wire the frontend to use the colormap from the downloaded "conditions" JSON object (default viridis)

Also: minor format changes after running "Sort Imports" and "Format Document" on Python files.

https://screenshot.googleplex.com/6oZDweTC5tPwTgk

### CODE REVIEW

File: src/planscape/config/conditions_config.py [hunk 1]
```diff
@@ -40,44 +40,32 @@ def check_regions(regionlist) -> bool:
...
 
         def check_metric(metric) -> bool:
             return (isinstance(metric, dict) and
-                    metric.keys() <= {'metric_name', 'filepath', 'display_name', 'current_conditions_only'} and
+                    metric.keys() <= {'metric_name', 'filepath', 'display_name', 'current_conditions_only', 'colormap'} and
                     isinstance(metric['metric_name'], str))
 
         return 'regions' in self._config and check_regions(self._config['regions'])
 
-    def get_regions(self) -> List[Region]:
```
Comment (elsiel23) [inline src/planscape/config/conditions_config.py:69]: are we intentionally deleting this?

File: src/interface/src/app/map/map.component.ts [hunk 2]
```diff
@@ -652,13 +652,19 @@ export class MapComponent implements AfterViewInit, OnDestroy {
     filepath = filepath.substring(filepath.lastIndexOf('/') + 1) + '.tif';
     console.log(filepath);
 
+    let colormap = map.config.dataLayerConfig.colormap;
+    if (filepath?.length === 0 || !filepath) {
```
Comment (thequeenofspades) [inline src/interface/src/app/map/map.component.ts:656]: We have a check for this above (line 648) so the function would have already returned at this point.

File: src/interface/src/app/types/data.types.ts [hunk 3]
```diff
@@ -3,24 +3,28 @@ export interface ConditionsConfig {
   filepath?: string;
   region_name?: string;
   pillars?: PillarConfig[];
+  colormap?: string;
```
Comment (thequeenofspades) [inline src/interface/src/app/types/data.types.ts:6]: Not in this PR, but for a follow-up: can we add a REST endpoint to get the actual color values associated with a colormap name? We'll need those to render the legend with the right color values.

**Review by thequeenofspades — APPROVED** [review]

File: src/interface/src/app/map/map.component.ts (hunk 2 above)
Comment (elsiel23) [inline src/interface/src/app/map/map.component.ts:656]: should this be colormap? same after the ||

File: src/interface/src/app/types/data.types.ts (hunk 3 above)
Comment (riecke) [reply src/interface/src/app/types/data.types.ts:6]: Will do!  Do you want them in hex RGB format?

File: src/planscape/config/conditions_config.py (hunk 1 above)
Comment (riecke) [reply src/planscape/config/conditions_config.py:69]: Looks like there was a merge conflict.   See below.

**Review by elsiel23 — APPROVED** [review]

File: src/interface/src/app/map/map.component.ts (hunk 2 above)
Comment (riecke) [reply src/interface/src/app/map/map.component.ts:656]: Yes.  I will fix.

**Pull request merged**